src/core/Text3D.tsx

```diff
@@ -0,0 +1,96 @@
+import { extend, Node } from '@react-three/fiber'
+import * as React from 'react'
+import { suspend } from 'suspend-react'
+import * as THREE from 'three'
+import { TextGeometry, FontLoader, TextGeometryParameters } from 'three-stdlib'
+
+extend({ RenamedTextGeometry: TextGeometry })
```

Comment:
could this be inlined into the component as a useMemo(() => extend(...), []) ? most drei components should be doing it already. otherwise it creates a side effect that prevents tree shaking.